docs/proposals/0001-cli-design.md

```diff
@@ -0,0 +1,49 @@
+# CLI design
+
+authors: @toumorokoshi, @zph
+
+
+This proposal outlines the design decisions and considerations for the Tome CLI.
+
+## Background
+
+Tome was initially designed as a drop-in replacement for [sub](https://github.com/basecamp/sub), to ensure portability.
```

Comment:
Is backwards compatibility a goal? Is that absolute compatibility or `generally strive to be compatible unless we have good reason otherwise`?

Reply:
I think this is ill-worded... It's never intended to be a "drop-in" replacement: just something that is similar enough to sub that it doesn't take substantial work to migrate over.

To your point, I'm arguing for compatibility to the latter (best effort). I'll update the doc.

Reply:
updated, please take a look.